src/substrait/type/Type.cpp

```diff
@@ -527,7 +527,9 @@ std::string ParameterizedMap::signature() const {
   sign << "<";
   sign << keyType()->signature();
```

Comment:
Do we need to worry about `keyType()` being `nullptr`?

Reply:
Check added.